pkg/controller/account/byoc.go

```diff
@@ -373,3 +372,8 @@ func validateBYOCClaim(accountClaim *awsv1alpha1.AccountClaim) error {
 
 	return nil
 }
+
+// Print a human readable error in the operator logs for debugging
+func printErr(location string, message error) {
+	fmt.Printf("\n+++++ In %s: %s +++++\n\n", location, message)
+}
```

Comment:
@clcollins can you please remove this debug function?

Reply:
@rogbas Removed. Sorry for looking over it the first time.